misc/helm-charts/operator/values.yaml

```diff
@@ -248,18 +249,18 @@ telemetry:
 
 # Network policies configuration
 networkPolicies:
-  # Whether to enable network policies for securing communication between pods
+  # -- Whether to enable network policies for securing communication between pods
   enabled: false
-  # internal communication between Materialize pods
+  # -- internal communication between Materialize pods
```

Comment:
? Does it need "Whether to enable internal communication ...."

Reply:
Good point! Just updated this!